--- a/Cargo.toml
+++ b/Cargo.toml
@@ -162,7 +162,10 @@
 cctk = { git = "https://github.com/pop-os/cosmic-protocols//", package = "cosmic-client-toolkit", rev = "32283d7" }
 cosmic-protocols = { git = "https://github.com/pop-os/cosmic-protocols//", rev = "32283d7" }
 
-# [patch."https://github.com/pop-os/libcosmic"]
+[patch."https://github.com/pop-os/libcosmic"]
+libcosmic = { git = "https://github.com/pop-os/libcosmic", rev = "8cb10bf6e24b86a7d2a9cf8c662d3e0154db1b21" }
+cosmic-config = { git = "https://github.com/pop-os/libcosmic", rev = "8cb10bf6e24b86a7d2a9cf8c662d3e0154db1b21" }
+cosmic-theme = { git = "https://github.com/pop-os/libcosmic", rev = "8cb10bf6e24b86a7d2a9cf8c662d3e0154db1b21" }
 # iced_core = { git = "https://github.com/pop-os/libcosmic//", branch = "theme-v2" }
 # iced_futures = { git = "https://github.com/pop-os/libcosmic//", branch = "theme-v2" }
 # libcosmic = { git = "https://github.com/pop-os/libcosmic//", branch = "theme-v2" }
