--- a/Cargo.toml
+++ b/Cargo.toml
@@ -147,14 +147,17 @@

 [workspace.dependencies.cosmic-config]
 git = "https://github.com/pop-os/libcosmic"
+rev = "8cb10bf6e24b86a7d2a9cf8c662d3e0154db1b21"
 default-features = false

 [workspace.dependencies.cosmic-theme]
 git = "https://github.com/pop-os/libcosmic"
+rev = "8cb10bf6e24b86a7d2a9cf8c662d3e0154db1b21"
 default-features = false

 [workspace.dependencies.libcosmic]
 git = "https://github.com/pop-os/libcosmic"
+rev = "8cb10bf6e24b86a7d2a9cf8c662d3e0154db1b21"
 default-features = false
 features = ["advanced-shaping"]
