--- a/daemon/src/lib.rs
+++ b/daemon/src/lib.rs
@@ -20,7 +20,7 @@
 
 impl Default for UserFilter {
     fn default() -> Self {
-        let login_defs_data = fs::read_to_string("/etc/login.defs").unwrap_or_default();
+        let login_defs_data = fs::read_to_string("/overlayer/syshub/etc/login.defs").unwrap_or_default();
         let login_defs = whitespace_conf::parse(&login_defs_data);
         Self {
             uid_min: login_defs
