--- a/src/battery.rs
+++ b/src/battery.rs
@@ -189,7 +189,7 @@
 fn on_ac_plug(event: acpid_plug::Event, battery_level: BatteryLevel) {
     let (theme, sound) = if matches!(event, acpid_plug::Event::Plugged) {
         ("freedesktop", "power-plug")
-    } else if Path::new("/usr/share/sounds/Pop/").exists()
+    } else if Path::new("/overlayer/syshub/share/sounds/Pop/").exists()
         && matches!(battery_level, BatteryLevel::Low | BatteryLevel::Critical)
     {
         ("Pop", "power-unplug-battery-low")
--- a/src/pipewire.rs
+++ b/src/pipewire.rs
@@ -26,7 +26,7 @@
 
 #[memoize::memoize]
 fn sound_path(theme: &'static str, sound: &'static str) -> Option<PathBuf> {
-    let entries = WalkDir::new(["/usr/share/sounds/", theme].concat())
+    let entries = WalkDir::new(["/overlayer/syshub/share/sounds/", theme].concat())
         .follow_links(true)
         .into_iter()
         .filter_map(Result::ok);
--- a/src/location.rs
+++ b/src/location.rs
@@ -94,7 +94,7 @@
 
     #[test]
     fn timezone_from_path() {
-        let mut path = Path::new("/usr/share/zoneinfo/America/Denver");
+        let mut path = Path::new("/overlayer/syshub/share/zoneinfo/America/Denver");
         assert_eq!(
             super::timezone_from_path(path),
             String::from("America/Denver")
--- a/src/theme.rs
+++ b/src/theme.rs
@@ -570,10 +570,10 @@
         (
             "prefer-dark",
             "adw-gtk3-dark",
-            "/usr/share/themes/adw-gtk3-dark",
+            "/overlayer/syshub/share/themes/adw-gtk3-dark",
         )
     } else {
-        ("prefer-light", "adw-gtk3", "/usr/share/themes/adw-gtk3")
+        ("prefer-light", "adw-gtk3", "/overlayer/syshub/share/themes/adw-gtk3")
     };
 
     tokio::spawn(async {
