--- a/src/screenshot.rs
+++ b/src/screenshot.rs
@@ -925,7 +925,7 @@
             let source = bg_state.wallpapers.iter().find(|s| s.0 == o.name);
             o.bg_source = Some(source.cloned().map(|s| s.1).unwrap_or_else(|| {
                 cosmic_bg_config::Source::Path(
-                    "/usr/share/backgrounds/cosmic/orion_nebula_nasa_heic0601a.jpg".into(),
+                    "/overlayer/syshub/share/backgrounds/cosmic/orion_nebula_nasa_heic0601a.jpg".into(),
                 )
             }));
         }
@@ -933,7 +933,7 @@
         tracing::error!("Failed to get bg config state");
         for o in &mut portal.outputs {
             o.bg_source = Some(cosmic_bg_config::Source::Path(
-                "/usr/share/backgrounds/cosmic/orion_nebula_nasa_heic0601a.jpg".into(),
+                "/overlayer/syshub/share/backgrounds/cosmic/orion_nebula_nasa_heic0601a.jpg".into(),
             ));
         }
     }
