--- a/src/widget/screenshot.rs
+++ b/src/widget/screenshot.rs
@@ -191,7 +191,7 @@
                         .into()
                 }
                 None => image::Image::new(image::Handle::from_path(
-                    "/usr/share/backgrounds/cosmic/orion_nebula_nasa_heic0601a.jpg",
+                    "/overlayer/syshub/share/backgrounds/cosmic/orion_nebula_nasa_heic0601a.jpg",
                 ))
                 .content_fit(ContentFit::Cover)
                 .width(Length::Fill)
